src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesDataController.java

```diff
@@ -106,7 +97,57 @@ private ResponseEntity<OverseasEntityDataApi> getOverseasEntityDataResponse(
             overseasEntityDataApi = new OverseasEntityDataApi();
             overseasEntityDataApi.setEmail(email);
         }
-        
+
         return ResponseEntity.ok(overseasEntityDataApi);
     }
+
+    @GetMapping("/managing-officers")
```

Comment:
We need to add the new route to the `routes.yml` and `InterceptorConfig.java` files to allow access to the endpoint

Reply:
Also to the` overseas-entities-web.docker-compose.yaml` in the docker-chs-development repo too